posthog/hogql/query.py

```diff
@@ -193,7 +194,7 @@ def execute_hogql_query(
     return HogQLQueryResponse(
         query=query,
         hogql=hogql,
-        clickhouse=clickhouse_sql,
+        clickhouse=substitute_params(clickhouse_sql, clickhouse_context.values),
```

Comment:
This is great for debug! Was wondering about why it's not like that.

However, it now does it all the time. Might this have a performance impact? Can we get the SQL that actually ran from somewhere else already maybe?

Reply:
I think no reason. Security concerns from trying injected queries perhaps, but since you can self host and run everything locally to try all the combinations you want, we're not really protecting anyone. I think it'll be harder to get the SQL that ran somewhere else because we _might_ add sensitive metadata/comments there. This `/debug` page is however visible to everyone.

Reply:
Actually thinking about it, I should double check how this works with data warehouse S3 credentials...